docs/DeveloperGuide.md

```diff
@@ -192,10 +193,10 @@ command:
 
 ```java
 @Override
-public DeleteCommand<T> createCommand(CommandParam param) throws ParseException{
-    int index = param.getUnnamedIntOrThrow(); // Look at here
-    return new DeleteCommand<>(index, getManagerFunction, deleteFunction);
-}
+public DeleteCommand<T> createCommand(CommandParam param)throws ParseException{
```

Comment:
Fix the autoformat here (can be disabled in Intellij Settings, under the languages and framework, find Markdown, and then make sure that this is unchecked. 
![IntelliJ IDEA-Settings-2023_04_10-18_28_35](https://user-images.githubusercontent.com/36021591/230884803-2698b197-d7ed-4965-874a-c39708788b1a.png)

Reply:
Done!